custom_components/thermal_comfort/sensor.py

```diff
@@ -393,7 +399,7 @@ def __init__(
         self._attr_extra_state_attributes = {}
         if self._device.unique_id is not None:
             self._attr_unique_id = id_generator(self._device.unique_id, sensor_type)
-        self._attr_should_poll = False
+        self._attr_should_poll = POLL_DEFAULT
```

Comment:
This poll doesn't have anything to do with the domain poll setting. We don't actually use this poll mode anymore since we can't set the scan_interval for it with the new yaml config.

Reply:
`poll` and `poll`. I see.

Reply:
I admit the current implementations it isn't really classic poll anymore since we push at a periodic time put it has basically the same effect. It's just the fastest solution i could think of for adding back scan_interval for the moment and make the poll option useful again. And i just pushed it out to have some working fix. Should i add some comments in the code to explain this better?